src/annotator/delegator.js

```diff
@@ -31,13 +31,18 @@ export default class Delegator {
    * @param {Object} [config]
    */
   constructor(element, config) {
-    // Some subclasses rely on a legacy mechanism of defining options at
-    // the class level by defining an `options` property on the prototype.
-    const classOptions = this.options || {};
-    this.options = { ...classOptions, ...config };
-    this.element = $(element);
+    this.options = { ...config };
+    this.element = element;
 
-    this.on = this.subscribe;
+    const el = /** @type {any} */ (element);
+    let eventBus = el._hypothesisEventBus;
```

Comment:
OK, so, in applied reality, basically everything that calls `Delegator`'s constructor passes the same `element`...is that true? So in most cases that element would already have an event bus...I'm just wondering if there's an easy way to make this feel less abstract...

Reply:
> OK, so, in applied reality, basically everything that calls Delegator's constructor passes the same element...is that true?

Everything that relies on publishing or subscribing to annotator's internal events must do, but instances of `Delegator` which don't care about events (I believe that's only `BucketBar`) don't have to.

> So in most cases that element would already have an event bus...I'm just wondering if there's an easy way to make this feel less abstract...

One possibility in future might be to refactor so that instead of inheriting from `Delegator`, classes which need the event bus would receive it explicitly as a constructor argument, so it wouldn't be tied to a DOM element. In this PR I'm just refactoring to remove the jQuery dependency but keeping the behavior the same as far as `Delegator` implementations/consumers are concerned.